tensorflow/python/ops/math_ops.py

```diff
@@ -330,6 +330,25 @@ def pow(x, y, name=None):
     return gen_math_ops._pow(x, y, name=name)
 
 
+def exp(x, name=None):
```

Comment:
@concretevitamin: crazy thought, but do you think it might be possible to do this within gen_*_ops autogeneration?  It seems unfortunate that we have to add this wrapper for every op in TensorFlow.